earthpy/spatial.py

```diff
@@ -1,12 +1,12 @@
-import contextlib
 import os
 import geopandas as gpd
```

Comment:
following proper order numpy, and os and contextlib should go at the top. Then matplotlib.  Then the spatil stuff then earthpy. please adjust the order @shwh2628  to follow conventions! geopandas is definitely considered a 3rd party package!